Workbooks/Ubiquiti.json

```diff
@@ -0,0 +1,330 @@
+{
+    "version": "Notebook/1.0",
+    "items": [
+      {
+        "type": 1,
+        "content": {
+          "json": "**NOTE**: This data connector depends on a parser based on Kusto Function **UbiquitiAuditEvent** to work as expected. Follow [these steps](https://aka.ms/sentinel-UbiquitiAuditEvent-parser) to get this Kusto Function"
```

Comment:
Please change parser link to same as in data connector
eg. https://aka.ms/sentinel-Ubiquiti-parser

Reply:
updated